### PR TITLE
Feature/home-scene

### PR DESCRIPTION
**Pull request by casperstorm**
This PR introduces a home-scene with a basic side menu. The idea is that we can use this as a starting point, and evovle where needed when we get furter:
screenshot 2018-11-06 at 14 47 42

### CODE REVIEW

**Comment by egeriis** [comment]
This is great!

I would like for us to consider a few design patterns before committing to a particular layout. Maybe it's time to draw up what features we'd like to have and how to lay them out?

**Comment by casperstorm** [comment]
@egeriis I agree!
How and where should we do this?

File: src/scenes/home/index.tsx [hunk 1]
```diff
@@ -1,33 +1,93 @@
...
-          trigger="click"
-          visible={this.state.visible}
-          onVisibleChange={this.handleVisibleChange}>
-          <Button type="primary">BOOST ME/ENGINE</Button>
-        </Popover>
-      </div>
+      <Layout>
+        <Sider
+          theme="dark"
+          style={{
```
Comment (kasperkronborg) [inline src/scenes/home/index.tsx:55]: Shouldn't this be in the CSS?

File: src/scenes/home/index.tsx [hunk 2]
```diff
@@ -1,33 +1,93 @@
...
+              <span>View</span>
+            </Menu.Item>
+            <Menu.Item key="2">
+              <Icon type="upload" />
+              <span>Upload</span>
+            </Menu.Item>
+          </Menu>
+        </Sider>
+        <Layout>
+          <Header style={{ background: '#fff', padding: 0 }}>
```
Comment (kasperkronborg) [inline src/scenes/home/index.tsx:78]: Shouldn't this be in the CSS?

File: src/scenes/home/index.tsx [hunk 3]
```diff
@@ -1,33 +1,93 @@
...
+            <Icon
+              className="trigger"
+              type={this.state.collapsed ? 'menu-unfold' : 'menu-fold'}
+              onClick={this.toggle}
+            />
+          </Header>
+          <Content
+            style={{
```
Comment (kasperkronborg) [inline src/scenes/home/index.tsx:86]: Shouldn't this be in the CSS?

**Review by kasperkronborg — APPROVED** [review]

**Comment by kasperkronborg** [comment]
Besides my comments above (which can be disregarded if it isn't possible to put the inline styling into CSS), I think this is awesome.

**Comment by egeriis** [comment]
@casperstorm Maybe when we can meet and catch up? My calendar is fairly open next week

**Comment by casperstorm** [comment]
> Besides my comments above (which can be disregarded if it isn't possible to put the inline styling into CSS), I think this is awesome.

~~Thanks @kasperkronborg – looking at it.~~

This made me realize, we should properly agree if we should use any CSS preprocessor. [Ant Design](https://ant.design/), which we are using, are using LESS, however I dont really mind. Any suggestions @steffenagger, @kasperkronborg  and @egeriis?

Also that should properly be added by its own PR so a low hanging fruit right here after we have discussed.

**Comment by egeriis** [comment]
@casperstorm I prefer styled-components nowadays

**Pull request merged**